ddp-workspace/projects/ddp-rgp/src/styles/_activity.scss

```diff
@@ -377,3 +377,44 @@ ddp-activity-dropdown-picklist-question select {
     padding: 0;
   }
 }
+
+/**
+ * Enrollment nested activities
+ */
+.instance {
+  .instance__divider {
```

Comment:
This CSS block is to style the new "divider" for "embedded nested activities". The `.instance:first-child` selector below is to hide the divider for the first instance.

Before:
![rgp_bug_divider_before](https://user-images.githubusercontent.com/5713833/116942045-d10a6000-ac3e-11eb-8e87-cbf2b9b0878a.png)

After:
![rgp_bug_divider_after](https://user-images.githubusercontent.com/5713833/116942059-d5cf1400-ac3e-11eb-9048-fe72bb3728ec.png)